src/NzbDrone.Core/MediaFiles/EpisodeImport/Aggregation/Aggregators/AggregateQuality.cs

```diff
@@ -19,55 +20,66 @@ public class AggregateQuality : IAggregateLocalEpisode
             _logger = logger;
         }
 
-        public LocalEpisode Aggregate(LocalEpisode localEpisode, bool otherFiles)
+        public LocalEpisode Aggregate(LocalEpisode localEpisode, DownloadClientItem downloadClientItem, bool otherFiles)
         {
-            var augmentedQualities = _augmentQualities.Select(a => a.AugmentQuality(localEpisode))
+            var augmentedQualities = _augmentQualities.OrderBy(a => a.Order)
+                                                      .Select(a => a.AugmentQuality(localEpisode, downloadClientItem))
                                                       .Where(a => a != null)
-                                                      .OrderBy(a => a.SourceConfidence);
+                                                      .OrderBy(a => a.SourceConfidence)
```

Comment:
Remove this. You can't sort by SourceConfidence and Order both. Sort by Order only and let the loop later sort out what the highest confidence is (it already is doing that).